modules/renter/hostdb/hostdb.go

```diff
@@ -107,7 +107,9 @@ func newHostDB(g modules.Gateway, cs modules.ConsensusSet, persistDir string, de
 	hdb.hostTree = hosttree.New(hdb.calculateHostWeight)
 
 	// Load the prior persistence structures.
+	hdb.mu.Lock()
 	err = hdb.load()
+	hdb.mu.Unlock()
```

Comment:
`load` spinning up goroutines makes me uneasy. A function called `load` should just do one thing: load data into the `hdb` object. You could avoid the locking by moving the call to `queueScan` into `newHostDB`, or perhaps `threadedScan`.

In the interest of getting the release out, I'll merge this, but I don't like it.